coffea/nanoevents/schemas/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .base import BaseSchema
```

Comment:
I'm not sure I see the motivation in splitting this module out at this stage. The file was not that large.
Did you make any other changes than just dividing the code into files?

Reply:
No - just thinking forward to automatic schema generators and stuff like that. If a random person wants to make their own schema it's nice to have it compartmentalized, no other distractions in the file, same for 'examples' with the TreeMaker and NanoAOD schemas.

Reply:
I agree it needs to happen eventually, just not sure about now (also a bit sad to lose the change history for it)
But didn't you need to make a change to implement #382

Reply:
I found it https://github.com/CoffeaTeam/coffea/blob/master/coffea/nanoevents/schemas/nanoaod.py#L128-L135 looks fine (though `type(self)` imo looks nicer than dunder, but it is to taste)